### ISSUE TITLE
amp-list not resized after page reload.

### ISSUE THREAD

**Issue by kborisov93** [ampproject/amphtml#28309]
## What's the issue?

If the user scrolls down the list and then reloads the page, the list is not automatically resized. 

## How do we reproduce the issue?
[https://somnishop.com/kategorie/schnarchschienen/](https://somnishop.com/kategorie/schnarchschienen/)
Scroll down the product list and reload the page.

## What browsers are affected?

All browsers.

## Which AMP version is affected?

2004252135000

**Comment by kborisov93** [comment]
Possible workarounds:

- To use the _changeToLayoutContainer_ action, but this will require a user interaction, which is absurd.

- To somehow prevent the page from auto scrolling, but I haven't found a way to do this in AMP.

**Comment by caroqliu** [comment]
Hi @kborisov93 thanks for filing this issue! I was able to reproduce the behavior you described. Has this feature worked as expected before this version?

**Comment by kborisov93** [comment]
No, it behaves like that for at least 2 months. I thought that I made some mistake, and got to it a few days ago.

**Comment by kborisov93** [comment]
Hi, @caroqliu do you have any progress on the issue and do you expect it to be fixed soon?
If not I will work on some workaround.

**Comment by caroqliu** [comment]
We probably do not have the bandwidth to take this on soon, unfortunately.

**Comment by jaygray0919** [comment]
bump
